File: Beispiele/local-variable-binding/DeclOrder333.java
// Scott 3.3.3
public final class DeclOrder333 {
    private DeclOrder333() { }

    private static int n = 1;

    private static void f() {
        int m = n;
        System.out.println(m);
        int n = 2;
        System.out.println(n);
    }

    public static void main(String[] args) {
        f();
        System.out.println(n);
    }
}

File: Beispiele/local-variable-binding/decl-order-3-3-3.c
// Scott 3.3.3
#include <stdio.h>
int n = 1;

void f() {
    int m = n;
    printf("%d\n", m);
    int n = 2;
    printf("%d\n", n);
}

int main(void) {
    f();
    printf("%d\n", n);
}

